OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -168,12 +168,28 @@ void addDeviceConsoleReporterToModel(Model& model, Device& device,
 } // namespace OpenSim
 
 
+void executeExample();
```

Comment:
I like `hopToIt()` :rabbit2:, but it isn't descriptive so we probably shouldn't encourage in a workshop example. `runHopperExample()` sounds good to me.